INSTALLATION.md

```diff
@@ -1,218 +1,156 @@
 
-2. Using a mongodb database installed on your local system natively :- Follow the setup guide on [mongodb docs](https://www.mongodb.com/docs/manual/administration/install-community/) for your respective operating system.
+# Clone this repository
 
-3. Using a containerized mongodb docker container on your local system :- Follow the setup guide using this [video tutorial](https://www.youtube.com/watch?v=uklyCSKQ1Po).
+First you need a local copy of talawa-api. Run the following command in a directory of choice on your local system.
 
-**Note:** If you are running mongodb on a remote system, either a cloud service or a system under your control, then ensure you have provided the correct access permissions and firewall openings for the VM/server/service where the mongodb is hosted.
+        git clone https://github.com/PalisadoesFoundation/talawa-api
 
-Paste the connection string for your database as the value for environment variable named `MONGO_DB_URL` inside the `.env` file present in the root directory of this application.
+This will download a local copy of talawa-api in that directory.
 
-# Talawa-api Installation
+Change your current directory in your terminal(shell) to the root directory of cloned repository. Use this command:-
 
-Talawa API is mainly written and built using `node.js` and `typescript`. We are providing some common approaches to set it up on your system.
+        cd ./talawa-api
 
-## Installation using the standard method
 
-Pre-requisites :-
+# Setting up node.js
 
-1. A running mongodb database connection string. Refer to `Setting Up The MongoDB Database` section for more information.
+Best way to install and manage node.js is making use of node managers. Two most popular node managers right now are [fnm](https://github.com/Schniz/fnm) and [nvm](https://github.com/nvm-sh/nvm). We'd recommend fnm because it's much faster than nvm. Install whichever one you want and follow their guide to set up node.js on your system.
 
-2. NodeJS installed on your local system(version 14 or above preferrable). It's best to make use of a node version manager to manage nodejs versions on your system. Two popular node version managers currently are [fnm](https://github.com/Schniz/fnm) and [nvm](https://github.com/nvm-sh/nvm). Follow the setup guide for either of them and install NodeJS.
 
-Follow these steps to get the API running :-
+**Note:** Make sure you are in the root directory of the talawa-api repository that you've cloned. The following instructions assume you're already in the root directory.
 
-1.  Clone this repo to your local machine :-
 
-        git clone https://github.com/PalisadoesFoundation/talawa-api
+# Creating .env file
```

Comment:
We should mention the important .env file parameters and what they mean. It's importan to let the user know what parameters will need to be changed beforehand and why. These were in the prior document and should be referenced.

- ACCESS_TOKEN_SECRET
- REFRESH_TOKEN_SECRET
- MONGO_DB_URL
- RECAPTCHA_SECRET_KEY
- MAIL_USERNAME
- MAIL_PASSWORD